runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsOverlay.java

```diff
@@ -249,6 +255,34 @@ private void renderTileTooltip(Graphics2D graphics, Tile tile)
 		}
 	}
 
+	private void renderMovementInfo(Graphics2D graphics, Tile tile)
+	{
+		Polygon poly = Perspective.getCanvasTilePoly(client, tile.getLocalLocation());
+
+		if (poly != null && poly.contains(client.getMouseCanvasPosition().getX(), client.getMouseCanvasPosition().getY()))
+		{
+			// We can only get CollisionMaps only for instanced regions.
```

Comment:
I believe we have reversed this behavior, so this should be available in non-instanced areas now.

Reply:
Ok. I removed the isInInstancedRegion() check below and tested it in a few non-instanced regions. It worked in all of the regions I tested so i'll keep that way.